app/assets/javascripts/settings.js

```diff
@@ -71,6 +71,8 @@ $(document).ready(function() {
 function onEnterEdit(item){
   item.removeClass("editable");
   item.tooltip('destroy');
+  var unescaped_value = _.unescape(item.find('input').val());
```

Comment:
When entering edit mode, the value parameter becomes to input tag, hence unescape is needed.